test/integration/simple_example/controls/gcloud.rb

```diff
@@ -13,8 +13,13 @@
 # limitations under the License.
 
 project_id = attribute('project_id')
-region = attribute('region')
 
-control "gcloud" do
-  title "Google Cloud configuration"
+control 'simple startup-script-custom' do
+  title "With the simple example of startup-script-custom calling stdlib::info and stdlib::cmd"
+
+  describe command("gcloud compute instances list --project #{project_id}") do
```

Comment:
This test is validating the test infrastructure rather than the behavior we care about. What do you think of changing this to take an output of the created VM, pull the VM description with `gcloud compute instances describe <instance>`, and testing the metadata values on it?

Reply:
I think that's a good additional test.

Could you help me understand the problem with this test as-is?  It's intended as the simplest end-to-end test for the module, validating a basic "hello world" from `startup-script-custom`.

Reply:
So to start off this is a very small concern, so I'm going to merge this PR in a moment regardless.

The best way to explain it is that with this test as-is, we could zero out all of the startup-scripts files or not even apply the startup script to this GCE instance, and this test would still pass. Nominally if we do something radical like that we'll see a corresponding test failure, but since the property we're testing is the instance name, we'll never see this test fail.

The behavior that's more relevant to us is information on the VM related to the startup-scripts module, and specifically the metadata on the GCE instance. If we do assertions that the GCE instance metadata has metadata containing the string `stdlib::debug` we'll have a test that'll actually fail if we zero out the startup-scripts files.

Rephrased for brevity, this test will almost always pass and may miss bugs; we could change this test to make it catch more bugs.

However this does provide value and we can do further iterations on this, so I'm going to accept this work as-is, and I can submit a followup PR to tweak this.

Reply:
Ah yeah I see now.  This specific example is cruft left over from getting the test suite up and running.  The actual end-to-end test follows in `its('stdout') { should match('INFO startup-script: Return code 0.') }`

I've removed the needless example in #15